pkg/virt-launcher/virtwrap/agent-poller/agent_poller.go

```diff
@@ -130,6 +130,28 @@ func (s *AsyncAgentStore) GetSysInfo() api.DomainSysInfo {
 	}
 }
 
+// GetInterfaceStatus returns the interfaces Guest Agent reported
+func (s *AsyncAgentStore) GetInterfaceStatus() []api.InterfaceStatus {
```

Comment:
The `Get` prefix is redundant, but I recognize it is consistent with the rest here.